.circleci/config.yml

```diff
@@ -60,6 +77,8 @@ jobs:
             python -m pip install mmcv-full -f https://download.openmmlab.com/mmcv/dist/cpu/torch<< parameters.torch >>/index.html
             python -m pip install mmdet
             python -m pip install -r requirements.txt
+            python -m pip uninstall pycocotools -y
+            python -m pip install pycocotools --no-binary pycocotools
```

Comment:
Need a line of comment to explain why

Reply:
I removed these lines since the numpy issue has been resolved by loose the version constraint of pycocotools.